RoaringBitmap/src/main/java/org/roaringbitmap/ArrayContainer.java

```diff
@@ -1125,10 +1125,20 @@ public int serializedSizeInBytes() {
   @Override
   public BitmapContainer toBitmapContainer() {
     BitmapContainer bc = new BitmapContainer();
-    bc.loadData(this);
+    copyBitmapTo(bc.bitmap, 0);
```

Comment:
So here, you are replacing `bc.loadData(this)` which calls...

```Java
  void loadData(final ArrayContainer arrayContainer) {
    this.cardinality = arrayContainer.cardinality;
    for (int k = 0; k < arrayContainer.cardinality; ++k) {
      final char x = arrayContainer.content[k];
      bitmap[(x) / 64] |= (1L << x);
    }
  }

```

With a call to 

```
 public void copyBitmapTo(long[] dest, int position) {
     for (int k = 0; k < cardinality; ++k) {
       final char x = content[k];
       final int i = position + (x) / 64;
       dest[i] |= (1L << x);
     }
   }
```

Why would that be faster? Can you explain?

Reply:
I expect that your motivation is to remove loadData to avoid code duplication, but I'd rather that your PR be minimal. If we are to remove loadData, then we should do it throughout the code base, including at 

https://github.com/RoaringBitmap/RoaringBitmap/blob/91ff377e89672170a5cda4b2ed0ec5c4ce1f21bd/RoaringBitmap/src/main/java/org/roaringbitmap/buffer/MappeableBitmapContainer.java#L1239

I suggest you leave loadData and do not change toBitmapContainer. Narrow down your optimization to the part you must have.

It might be fine to remove loadData, but I suggest doing it in a separate PR.

Code duplication is ok. We have a fair amount of code duplication.

Reply:
Yes, it was to avoid duplication. I'll put it back.